crates/biome_js_analyze/tests/specs/complexity/noUselessFragments/issue_6648.jsx

```diff
@@ -0,0 +1,4 @@
+<>
```

Comment:
add our magic comment that checks no diagnostics were emitted

```suggestion
/* should not generate diagnostics */
<>
```